.circleci/config.yml

```
@@ -722,14 +722,11 @@ jobs:
           pattern: '^mysqldb_contrib-.*-mysqlclient'
 
   pymysql:
-    <<: *contrib_job
-    docker:
-      - image: *ddtrace_dev_image
-      - *mysql_server
+    <<: *machine_executor
     steps:
-      - run_tox_scenario:
-          wait: mysql
-          pattern: '^pymysql_contrib-'
+      - run_test:
+        pattern: 'pymysql$'
+        docker_services: "mysql"
```

```suggestion
          pattern: 'pymysql$'
          docker_services: "mysql"
```

This should fix the ci build error: https://app.circleci.com/pipelines/github/DataDog/dd-trace-py/18764/workflows/bf5a1291-5141-48d9-85d3-12dea77ec53a/jobs/1269979